docs/recipes/pug.md

```diff
@@ -34,27 +34,47 @@ $ npm install --save-dev gulp-pug
 Add this task to your `gulpfile.js`, it will compile `.pug` files to `.html` files in `.tmp`:
 
 ```js
-gulp.task('views', () => {
-  return gulp.src('app/*.pug')
+function pug() {
+  return src('app/*.pug')
     .pipe($.plumber())
     .pipe($.pug({pretty: true}))
-    .pipe(gulp.dest('.tmp'))
-    .pipe(reload({stream: true}));
-});
+    .pipe(dest('.tmp'))
+    .pipe(server.reload({stream: true}));
+}
 ```
 
 We are passing `pretty: true` as an option to get a nice HTML output, otherwise Pug would output the HTML on a single line, which would break our comment blocks for wiredep and useref.
 
-### 3. Add `views` as a dependency of both `html` and `serve`
+### 3. Add `pug` task to `server` and `build` process
 
 ```js
-gulp.task('html', ['views', 'styles', 'scripts'], () => {
+if (isDev) {
+  serve = series(clean, pug, parallel(styles, scripts, fonts), startAppServer);
+} else if (isTest) {
+  serve = series(clean, pug, scripts, startTestServer);
+} else if (isProd) {
+  serve = series(build, pug, startDistServer);
+}
     ...
 ```
 
 ```js
-gulp.task('serve', ['views', 'styles', 'scripts', 'fonts'], () => {
-    ...
+const build = series(
+  clean,
+  parallel(
+    lint,
+    series(pug, parallel(styles, scripts), html),
+    images,
+    fonts,
+    extras
+  ),
+  addInlineAttr,
+  inlineCss,
+  injectFavicon,
+  sw,
```

Comment:
Seems like `addInlineAttr`, `inlineCss`, `injectFavicon`, and `sw` might be accidental leftovers from your personal setup? We don't have these tasks.